tests/api/v1/test_exception_handlers.py

```diff
@@ -15,6 +15,7 @@ def mock_config_db_disabled():
     yield
     config.config.database.ENABLED = db_enabled
 
+
```

Comment:
It looks like all the changes in this file are just formatting fixes? I appreciate these, but unless they're causing CI failures they should be removed from this PR in favor of a later PR that actually touches the logic here.

Reply:
hmmm. they came up at least as warnings for me in my formatting check locally. so i thought they may cause CI failures? i certainly wouldn't have changed otherwise. i suppose i can revert and we can see what happens to the CI jobs.

Reply:
Are you able to `git restore` this file?

Reply:
yup, have that locally with a few other minor updates, just trying to re-run tests and CI checks and it's _slow_